mcu-util/src/orb/main_board.rs

```diff
@@ -366,43 +333,93 @@ impl MainBoard {
                 .await
             {
                 Ok(CommonAckError::Success) => {
-                    info!("LEDs disabled");
+                    info!("White/booster LEDs turned off");
```

Comment:
so booster & white are the same LEDs on Diamond?

Reply:
Yes, I think just the naming is a bit confusing. In control-api we call them "white leds", it's the same leds. But also at the same time rgbleds can be just white.